brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1012 Improvement: Call to Action - Additional Button Icon Spacing is responsive control.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1012

### Screenshots

- https://share.getcloudapp.com/nOuXkN2O

### Types of changes




- Improvement.

### How has this been tested?

- Add CTA block, Enable Additional button.
- Add Icon to the button.
- You will see the Icon spacing option with responsive support.
- Check Icon spacing with before/after icon position on responsive devices.

### Checklist:
